docs/reference/env-variables.rst

```diff
@@ -0,0 +1,58 @@
+.. meta::
+    :description: rocBLAS environment variables
+    :keywords: AMD, rocBLAS, environment variables, environment
+
+.. _rocblas-envvariables:
+
+********************************************************************
+Environment variables
+********************************************************************
+
+The rocBLAS environment variables are collected in the following
+tables.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 30, 15, 20, 35
+
+    * - **Environment variable**
+      - **Default value**
+      - **Links**
+      - **Value**
+
+    * - | ``ROCBLAS_USE_HIPBLASLT``
+        | Provide manually control which GEMM backend is used.
+      - Unset by default.
+      - :ref:`Control the GEMM backend <rocblas-tensile_hipblaslt>`
+      - | **Unset**: GEMM backend is automatically selected.
+        | **0**: Tensile is always used as the GEMM backend.
+        | **1**: hipBLASLt is preferred as the GEMM backend, but will fallback to Tensile on problems for which hipBLASLt does not provide a solution or when errors are encountered using the hipBLASLt backend.
+
+    * - | ``ROCBLAS_DEVICE_MEMORY_SIZE``
+        | Set how much memory to preallocate.
+      - Unset by default.
+      - :ref:`rocblas_device_memory_size`
+      - | **0 or unset**: Lets rocBLAS manage the device memory.
+        | **Bigger than 0**: Sets the default handle device memory size to the specified size (in bytes).
+
+    * - | ``ROCBLAS_DEFAULT_ATOMICS_MODE``
+        | Set the default atomics mode during the creation of ``rocblas_handle``.
+      - Unset by default.
+      - :ref:`Atomic Operations`
+      - | **0**: Sets the default to :any:`rocblas_atomics_not_allowed`
+        | **1**: Sets the atomics to :any:`rocblas_atomics_allowed`
```

Comment:
Keep prefix "Sets the default ...." for both cases 0 and 1, as that should be repeated